src/deprecations.jl

```diff
@@ -341,10 +341,10 @@ function FacetValues(
 end
 
 function value(ip::Interpolation, ξ::Vec)
-    throw(DeprecationError("value(ip::Interpolation, ξ::Vec)" => "[reference_shape_value(ip, ξ, i) for i in 1:getnbasefunctions(ip)]"))
+    throw(DeprecationError("value(ip::Interpolation, ξ::Vec)" => "[reference_shape_value(ip, ξ, i) for i in 1:getnbasefunctions(ip)] or reference_shape_values!(N, ip, ξ) with preallocated N"))
```

Comment:
(Same for derivative below)

The custom highlighting doesn't really match here, so the simplest option would be
```suggestion
    throw(DeprecationError("value(ip::Interpolation, ξ::Vec) is deprecated, use `[reference_shape_value(ip, ξ, i) for i in 1:getnbasefunctions(ip)]` or `reference_shape_values!(N, ip, ξ)` (with preallocated N) instead"))
```

Reply:
Okay was a bit more tricky to recover styling.

Reply:
Yes, I didn't suggest having the styling (thinking no styling is better than wrong styling) since that seemed to be too much work... So sorry for creating the extra work!

Reply:
No worries! :)